boards/nrf51dk/src/io.rs

```diff
@@ -48,3 +47,59 @@ macro_rules! println {
         ($fmt:expr) => (print!(concat!($fmt, "\n")));
             ($fmt:expr, $($arg:tt)*) => (print!(concat!($fmt, "\n"), $($arg)*));
 }
+use core::fmt::{Write, write};
+use core::fmt::Arguments;
```

Comment:
This will probably fail rustfmt (from the root directory, run `make format`) because the use directives need to be at the top of the file.